### PR TITLE
Private Changes into dev

### PR DESCRIPTION
**Pull request by sacherjj**
Merging security updates into dev.

### CODE REVIEW

File: execution_engine/CHANGELOG.md [hunk 1]
```diff
@@ -16,11 +16,35 @@ All notable changes to this project will be documented in this file.  The format
...
 ### Changed
 * Change contract runtime to allow caching GlobalState changes during execution of a single block.
 * Fix some integer casts.
 * Change both genesis and upgrade functions to write `ChainspecRegistry` under the fixed `Key::ChainspecRegistry`.
+* Lift the temporary limit of the size of individual values stored in global state.
+* Lift the temporary limit of the global maximum delegator capacity.
+
+
+
+## 1.5.0
```
Comment (mssteuer) [inline execution_engine/CHANGELOG.md:30]: This is the wrong version number. It should be 1.4.5

Comment (sacherjj) [reply execution_engine/CHANGELOG.md:30]: We need to integrate the 1.4.5 changes into the CHANGELOG.md, but this is 1.5 code.

**Review by Fraser999 — APPROVED** [review]

File: execution_engine/CHANGELOG.md (hunk 1 above)
Comment (Fraser999) [reply execution_engine/CHANGELOG.md:30]: I think it's more down to the internal crates' versions no longer being tied to the protocol version of the release except for `casper-node`.  So, as part of the 1.4.5 release, the `casper-execution-engine` crate is being upgraded to 1.5.0.

**Comment by sacherjj** [comment]
bors merge

**Comment by sacherjj** [comment]
bors merge

**Pull request merged**